### DISCUSSION TITLE
Can't find a way to stop `FlatESLint` read ignore file

### DISCUSSION THREAD

**Discussion by fisker** [eslint/eslint#16335] [General] [answered]
In the new `FlatESLint` there seems no way to prevent reading `.eslintignore` file?
Am I missing something?  

Possible workaround for now `ignorePath: 'a-empty-file'`.

**Comment by mdjermanovic** [comment, marked as answer]
Did you try with `ignore: false`? (constructor option [ignore](https://github.com/eslint/eslint/blob/2c152ff0fb709b99e62c19ecd2c95689efacbe4c/lib/eslint/flat-eslint.js#L79))

Anyway, we plan to drop support for `.eslintignore` and .gitignore-style patterns in `FlatESLint`: https://github.com/eslint/eslint/issues/16264

**Reply by fisker** [reply]
It works. Thanks, I must be blind..

**Reply by fisker** [reply]
It seems that `ignore: false` makes ESLint ignore `ignores` in config too.

**Reply by nzakas** [reply]
ignore: false turns off all ignores, including those in the config file. You may be able to try ignorePath:null.
